lib/vue-simple-suggest.vue

```diff
@@ -320,13 +334,14 @@ export default {
 
         if (!this.hovered) {
           item = this.selected || this.suggestions[listEdge]
+          this.hover(item, undefined, this.getId(item, listEdge))
```

Comment:
This section is kinda cognitively bloated. Can be simplified a bit with a simple variable assignment, like it's done here with `item`.

Reply:
Will do.